Tests/mpc-kit-swiftTests/mpc_kit_swiftTests.swift

```diff
@@ -83,6 +84,13 @@ func mockLogin2 (email:String) throws -> String {
 
             let payload = TestPayload(subject: SubjectClaim(stringLiteral: subject), expiration: ExpirationClaim(value: modifiedDate), audience: "torus-key-test", isAdmin: false, emailVerified: true, issuer: "torus-key-test", iat: IssuedAtClaim(value: Date()), email: email)
             let jwt = try signers.sign(payload)
+            let parsedToken = parseToken(jwtToken: jwt);
```

Comment:
dont add parseToken function test here
write a test function and call mockLogin2 and parseToken there